src/components/MovieCard/MovieCard.js

```diff
@@ -0,0 +1,104 @@
+import styled from 'styled-components/macro'
+
+import Button from '../Button/Button'
+
+import 'css.gg/icons/css/film.css'
+
+export default function MovieCard({
+  onHandleDelete,
+  key,
+  movieName,
+  movieFoodCategory,
+  display,
+  right,
+}) {
+  return (
+    <>
+      <MovieWrapper key={key}>
```

Comment:
You don't have to pass the key. It's enough to give it your component in the map. 
```suggestion
      <MovieWrapper>
```